### PR TITLE
Test coverage for `Client` ops

### PR DESCRIPTION
**Pull request by cthulhu-rider**
- [x] accounting (#636)
- [x] container
- [x] netmap
- [x] reputation
- [x] session
- [x] object
- [ ] benchmarks (wontfix, #639)

### CODE REVIEW

**Comment by cthulhu-rider** [comment]
that's it for now :sweat_smile:

test code may look bulky, but now it's quite detective. The maintenance is expected to be easier in the future

testing allowed to revise many things. I've changed/fixed some of them right away. For less obvious and/or requiring more changes I opened several issues

File: container/container.go [hunk 1]
```diff
@@ -155,7 +155,7 @@ func (x *Container) readFromV2(m container.Container, checkFieldPresence bool) e
 
 		val = attrs[i].GetValue()
 		if val == "" {
-			return fmt.Errorf("empty attribute value %s", key)
+			return fmt.Errorf("empty value of the attribute %s", key)
```
Comment (roman-khimov) [inline container/container.go:158]: `empty %q attribute value` would be shorter.

**Review by roman-khimov — APPROVED** [review]
I can't review test changes here, sorry. Functional ones are OK.

File: client/object_hash.go [hunk 2]
```diff
@@ -153,7 +153,8 @@ func (c *Client) ObjectHash(ctx context.Context, containerID cid.ID, objectID oi
 
 	resp, err := c.object.GetRangeHash(ctx, req.ToGRPCMessage().(*protoobject.GetRangeHashRequest))
 	if err != nil {
-		return nil, rpcErr(err)
+		err = rpcErr(err)
+		return nil, err
```
[the remark this answers is not in the document]
Comment (cthulhu-rider) [reply client/object_hash.go:157]: sure, fixed

File: container/container.go (hunk 1 above)
Comment (cthulhu-rider) [reply container/container.go]: agree, changed

**Comment by cthulhu-rider** [comment]
> I can't review test changes here, sorry

np, can be done at any time

**Review by carpawell — APPROVED** [review]
Test commit was not viewed carefully.

**Pull request merged**